File: Practice problem/sum of the sequence in range/sum of the sequence in range.cs
// 연습문제 두 정수 사이의 합 Level 1

public class Solution
{
    public long add(int a, int b)
    {
        long result = 0;
        for (int i = a; i <= b; i++)
        {
            result += i;
        }
        return result;
    }
    public long solution(int a, int b)
    {
        long answer = 0;
        if (a == b) { answer = a; }
        else if (a < b) { answer = add(a, b); }
        else { answer = add(b, a); }
        return answer;
    }
}

File: Practice problem/sum of the sequence in range/sum of the sequence in range.py
# 연습문제 두 정수 사이의 합 Level 1

def add(a, b):
    result = 0
    for k in range(a, b+1):
        result = result + k
    
    return result

def solution(a, b):
    answer = 0
    if a == b:
        answer = a
    elif a < b:
        answer = add(a, b)
    else:
        answer = add(b, a)
    return answer